grpc-ballerina/tests/20_unary_client_for_anonymous_service.bal

```diff
@@ -16,19 +16,30 @@
 
 listener Listener ep20 = new (9110);
 
-service object {} AnonService = @ServiceDescriptor {descriptor: ROOT_DESCRIPTOR_20, descMap: getDescriptorMap20()} 
+service object {} AnonService1 = @ServiceDescriptor {descriptor: ROOT_DESCRIPTOR_20, descMap: getDescriptorMap20()} 
 service object {
 
-    remote isolated function hello(string value) returns string|error {
-        return "Hello Ballerina";
-    }
-};
+            remote isolated function hello1(string value) returns string|error {
+                return "Hello Ballerina";
+            }
+            remote isolated function hello2(string value) returns string|error {
+                return "Hello Ballerina";
+            }
+        };
+
+service object {} IncompleteService = @ServiceDescriptor {descriptor: ROOT_DESCRIPTOR_20, descMap: getDescriptorMap20()} 
+service object {
+
+            remote isolated function hello1(string value) returns string|error {
+                return "Hello Ballerina";
+            }
+        };
 
 service object {} unregisteredService = @ServiceDescriptor {descriptor: ROOT_DESCRIPTOR_20, descMap: getDescriptorMap20()} 
 service object {
 
-    remote isolated function hello1(string value) returns string|error {
-        return "Hello Ballerina";
-    }
-};
+            remote isolated function hello1(string value) returns string|error {
```

Comment:
Are the extra tabs intended?

Reply:
Fixed in here https://github.com/ballerina-platform/module-ballerina-grpc/pull/314/commits/a256abda484113d107c009da6123d63612a949cb, seems like a formatting issue. Manually fixed it.